packages/editor/CodeMirror/markdown/markdownReformatter.test.ts

```diff
@@ -144,34 +144,24 @@ describe('markdownReformatter', () => {
 		expect(tabsToSpaces(state, '  \t  ')).toBe('      ');
 	});
 
-	it('should correctly renumber a list with multiple selections', async () => {
-		const firstListText = [
+	it('should correctly renumber a list with multiple selections in that list', async () => {
+		const listText = [
```

Comment:
This test fails with the new version of CodeMirror. Testing list renumbering with multiple cursors manually, however, works.

I suspect that the syntax tree for ordered lists isn't loading correctly (or isn't being forced to load).